package.json

```diff
@@ -29,7 +30,8 @@
   "scripts": {
     "validate": "tsx ./bin/cli.ts validate",
     "generate": "tsx ./bin/cli.ts generate",
-    "generate:ci": "yarn generate --datadir ./data --outfile optimism.tokenlist.json",
+    "generate-default-tokens": "tsx ./bin/cli.ts generate-default-tokens",
+    "generate:ci": "yarn generate --datadir ./data --outfile optimism.tokenlist.json && yarn generate-default-tokens --datadir ./data --outfile default.tokens.json",
```

Comment:
unrelated to this pr but I think we made the script way too general.   It doesn't need this datadir prop or this outfile prop those could just be hardcoded

Reply:
agreed